docs/reference/python-api.rst

```diff
@@ -52,7 +52,7 @@ Now that the experiment has completed, you can grab the top-performing checkpoin
 .. _python-api-reference:
```

Comment:
can you also rename this `python-api-reference` reference everywhere?

Reply:
done